styleguide/_themes/derek/scss/globalElements/mainnav.scss

```diff
@@ -365,6 +368,25 @@
     }
   }
 
+  .dropdown-toggle {
+
+    &::after {
+      content: '\f0da';
+      font-family: FontAwesome;
+      margin-left: 10px;
+    }
+  }
+
+  .navbar-dropDown-triggerActive {
+    background-color: $gray-darker;
```

Comment:
Can you set this background color on `:focus` and `:hover` as well? Bootstrap is setting this to transparent.